### DISCUSSION TITLE
Any plan to upgrade spring boot to version 3.X ?

### DISCUSSION THREAD

**Discussion by tejashkasalkar-druck** [cloudfoundry/uaa#2517] [Q&A]
OSS support for SpringBoot 2.7.X ends in Nov 2023. Any plans to bump springboot dependency ?

**Comment by strehle** [comment]
see here we will update uaa soon,
https://github.com/cloudfoundry/uaa/issues/2115

**Reply by strehle** [reply]
We have identified some projects we need to either remove or fork in order to get spring boot 3 , therefore created https://github.com/orgs/cloudfoundry/projects/42
